waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -784,9 +787,19 @@ proc mountRlnRelayDynamic*(node: WakuNode,
     rlnIndex = regIndexRes.value
     debug "peer is successfully registered into the membership contract"
   else:
+    info "Peer is already registered to the membership contract"
     keyPair = memKeyPair.get()
     rlnIndex = memIndex.get()
 
+  var
+    rlnMembershipCredentials = RlnMembershipCredentials(membershipKeyPair: keyPair, rlnIndex: rlnIndex)
+
+  #Write RLN credentials
+  writeFile(RLN_CREDENTIALS_FILEPATH, pretty(%rlnMembershipCredentials))
+
+  #Since the files are stored as a raw text file, it is highly susceptible to theft.
+  #The files needs some encryption to resolve this.
```

Comment:
```suggestion
  # Since the files are stored as raw text file, it is highly susceptible to theft.
  # The files need some encryption to resolve this.
```

Reply:
Suggestion Incorporated